### ISSUE TITLE
Context API driven version of Drizzle

### ISSUE THREAD

**Issue by jeffscottward** [ConsenSys-archive/drizzle-legacy#48]
This blog has been making the rounds as of recent.
https://medium.freecodecamp.org/replacing-redux-with-the-new-react-context-api-8f5d01a00e8c
They actually got it working with the Redux timeline debugger if you scroll down too.

It would be great to unbundle from Redux if we are infact moving into a redux-less future and into one that leans on the Context Api for React. Redux is one level out from React so not sure if this feature request makes sense here. Maybe this repo could be split off into Drizzle-Redux? Maybe it doesn't matter when you really look at the guts of it?

Thoughts?
TLDR; I'm starting to feel "over" redux in my react apps. Maybe this library doesn't have to change at all though?

**Comment by soundyogi** [comment]
As I understand drizzle makes heavy use of sagas which are redux dependent.
The new context api does not replace redux aswell if you ask me. 

For small apps context api can be enough.
For larger ones I did not want to loose redux myself.
/2cents

**Comment by soundyogi** [comment]
Hi,

redux works with more libraries than react - locking all the state management to react is not a wise choice I think.

In fact drizzle-react uses the react context api to connect the drizzle redux store (which is quite a nice pattern to create specialized redux stores if you ask me)
https://github.com/trufflesuite/drizzle-react/blob/master/src/DrizzleContext.js

this issue can be closed

**Comment by DiscRisknBisque** [comment]
@jeffscottward , for an example of using Drizzle with the new context API, check out this tutorial: https://truffleframework.com/tutorials/getting-started-with-drizzle-and-react
